services/core-web/src/components/mine/Permit/PermitConditionCategory.tsx

```diff
@@ -66,7 +66,9 @@ export const EditPermitConditionCategoryInline = (props: IPermitConditionCategor
   return (
     <FormWrapper scrollOnToggleEdit={false} name={formName} onSubmit={handleSubmit} initialValues={props.category} isEditMode={isEditMode}>
       <Row className="flex" style={{ gap: '0.5em' }}>
-        <Field name="step" component={RenderField} required={true} validate={[required]} />
+        <Form.Item style={{ marginRight: 0, maxWidth: '40px' }}>
+          <Field name="step" component={RenderField} required={true} validate={[required]} />
```

Comment:
The Field component does already have a Form.Item within it, so I really don't think there should be another one wrapping it. Does it want to be a div? A Col? Probably a Col because it's a child of Row and Row likes all its direct children to be Cols. I did something similar on my branch, and to save you looking up the antd props, the Row probably wants `wrap={false}`

Reply:
Updated. Kept the wrap so it wraps nicely when squished. Kept the buttons in the same column so they wrap together